spec/models/replace_encryption_certificate_event_spec.rb

```diff
@@ -93,6 +103,19 @@ def certificate_created_with(params = {})
     expect(event.errors[:certificate]).to eq [t('certificates.errors.expires_soon')]
   end
 
+  it 'must not validate expiry when uploaded a admin' do
```

Comment:
must not validate expiry when uploaded by an admin

Reply:
fixed :)